pom.xml

```diff
@@ -112,6 +112,81 @@
             <artifactId>jackson-datatype-jsr310</artifactId>
             <version>2.12.5</version>
         </dependency>
+        <!-- Token Management (e.g., PKCS12, hardware tokens) -->
+        <dependency>
+            <groupId>eu.europa.ec.joinup.sd-dss</groupId>
+            <artifactId>dss-token</artifactId>
+            <version>6.2.RC1</version>
+        </dependency>
+
+        <!-- Apache Commons Utilities -->
+        <dependency>
+            <groupId>eu.europa.ec.joinup.sd-dss</groupId>
+            <artifactId>dss-utils-apache-commons</artifactId>
+            <version>6.2.RC1</version>
+        </dependency>
+
+        <!-- Document Management -->
+        <dependency>
```

Comment:
Some dependencies may already be included by others, so we might not need to import them separately. Could you please confirm whether all of them need to be added?